packages/altair-app/src/app/modules/altair/components/account-dialog/account-dialog.component.spec.ts

```diff
@@ -1,24 +1,39 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { describe, expect, it } from '@jest/globals';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { TranslateModule } from '@ngx-translate/core';
 import { mount, NgxTestWrapper } from '../../../../../testing';
 import { SharedModule } from '../../modules/shared/shared.module';
 
 import { AccountDialogComponent } from './account-dialog.component';
 
 describe('AccountDialogComponent', () => {
   let wrapper: NgxTestWrapper<AccountDialogComponent>;
+  let component: AccountDialogComponent;
+  let fixture: ComponentFixture<AccountDialogComponent>;
 
   beforeEach(async () => {
     wrapper = await mount({
       component: AccountDialogComponent,
       declarations: [AccountDialogComponent],
       providers: [],
-      imports: [SharedModule],
+      imports: [SharedModule, TranslateModule.forRoot(),],
       schemas: [NO_ERRORS_SCHEMA],
     });
+    fixture = TestBed.createComponent(AccountDialogComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(wrapper.component.nativeElement).toMatchSnapshot();
   });
+
+  it('should handle submitLogin', () => {
+    jest.spyOn(component.handleLoginChange, 'emit');
+
+    component.submitLogin();
+
+    expect(component.handleLoginChange.emit).toHaveBeenCalled();
```

Comment:
instead of using `jest.spyOn`, we could trigger the click event on the submit button (or submit event on the form) and check that the `handleLoginChange` event was emitted. For example, https://github.com/altair-graphql/altair/blob/00353c9ba59c24a412fb0e1ee60ef7433dfd62cc/packages/altair-app/src/app/components/query-collections/query-collections.component.spec.ts#L206-L222